doc/src/pgRouting-installation.rst

```diff
@@ -140,11 +154,12 @@ Dependencies
 
 .. rubric:: Compilation Dependencies
 
-To be able to compile pgRouting, make sure that the following dependencies are met:
+To be able to compile pgRouting, make sure that the following dependencies are 
+met:
 
-* C and C++0x compilers
-  * Compiling with Boost 1.56 up to Boost 1.74 requires C++ Compiler with C++03 or C++11 standard support
-  * Compiling with Boost 1.75 requires C++ Compiler with C++14 standard support
+* C and C++0x compilers - Compiling with Boost 1.56 up to Boost 1.74 requires 
+  C++ Compiler with C++03 or C++11 standard support - Compiling with Boost 1.75 
```

Comment:
Something is wrong with the indentation, it is unreadable:
https://github.com/pgRouting/pgrouting/blob/464e163343b800065df0c4a4a3f0e6230753ada2/doc/src/pgRouting-installation.rst#dependencies
Please check how in sphinx lists with indentations are done

Reply:
Will fix it right away.